src/index.js

```diff
@@ -44,13 +44,18 @@ define('pep-p', {
 
   // Restores the original children
   restoreChildren() {
-    this.innerHTML = '';
+    this.innerHTML = ''; // clear out all the children
+    // Create clones of the original children and put them back.
     this.originalChildNodes.forEach(child => this.appendChild(child.cloneNode(true)));
   },
 
   // render by updating the trimmed text to match the current size.
   render() {
+    // Clear the tooltip.
     this.removeAttribute('tooltip');
+    // Find the minimum height needed to display text.
+    this.innerHTML = 'AEIOU';
```

Comment:
I'd recommend including letters like `y` for descenders and `h` which may ascend higher than most capital letters. And possibly something with an accent or diacritical mark like `À`

Reply:
I realize it's going to vary by font how these are treated, but it may offer better coverage for height detection.

Reply:
Good idea! Updated.